.gitignore

```diff
@@ -1,2 +1,4 @@
 *__pycache__
 *.pytest_cache
+.coverage
+htmlcov/
```

Comment:
Is there any reason for `.coverage` to not have the `/`and `htmlcov` to have ?

Reply:
.coverage is a file, not a folder.